src/test/java/tests/QualifierTypeTests.java

```diff
@@ -0,0 +1,40 @@
+package tests;
+
+import static filter.expression.QualifierType.*;
+import static junit.framework.TestCase.*;
+
+import java.util.Optional;
+import java.util.Set;
+
+import org.junit.Test;
+
+public class QualifierTypeTests {
+
+    @Test
+    public void testCompletionKeywords() {
+
+        Set<String> keywords = getCompletionKeywords();
+
+        // Most qualifiers are represented
+        assertTrue(keywords.contains("sort"));
+
+        // Some qualifiers are deliberately excluded
+        assertFalse(keywords.contains("is"));
+
+        // Other non-qualifier keywords are included
+        assertTrue(keywords.contains("repo"));
+    }
+
+    @Test
+    public void testParse() {
```

Comment:
Can we have some alias parse test as well? To ensure that `resolveAlias()` is called.